frontend/catalyst/third_party/oqd/src/oqd_device_parameters.toml

```diff
@@ -0,0 +1,744 @@
+# OQD Device Parameters
```

Comment:
Just confirming, this is the prototype database with placeholder numbers right?

Reply:
Where all the data fields are filled in, these are the numbers that OQD sent over to us, so I think we can assume that these are the actual experiment parameters (although they are subject to change). There are a number of parameters with missing data since OQD hasn't finalized them yet; these are placeholders for now.